kube/smoke-test-list.yaml

```diff
@@ -0,0 +1,64 @@
+---
+kind: List
+apiVersion: v1
+metadata: {}
+items:
+- apiVersion: v1
+  kind: Pod
+  metadata:
+    name: origin-web-console-smoke-test-pod
+    labels:
+      app: origin-web-console-smoke-test
+    annotations:
+      prometheus.io/scrape: "true"
+      # todo: enable HTTPS
+      prometheus.io/scheme: http
+  spec:
+    restartPolicy: Never
+    containers:
+    - name: origin-web-console-smoke-test
+      image: benjaminapetersen/origin-web-console-smoke-test:latest
+      ports:                              
+      - containerPort: 3000
+        protocol: TCP
+      imagePullPolicy: IfNotPresent
+      env:
+      - name: CONSOLE_URL
+        value: https://192.168.100.3:8443
+- kind: Service
+  apiVersion: v1
+  metadata:
+    name: origin-web-console-smoke-test-service
+  spec:
+    ports:
+    - name: web
+      protocol: TCP
+      port: 3000
+      targetPort: 3000
+      nodePort: 0
+    selector:
+      app: origin-web-console-smoke-test
+    type: ClusterIP
+    clusterIP: 172.30.136.123
```

Comment:
This looks good.  Curious about `clusterIP`, I'm guessing this is one item we may have to update before deployment each time, similar to the above `CONSOLE_URL`?

Reply:
so the `clusterIP` will set the IP for the service but if we use it for online, there is a good chance that its going to be used... will probably decouple the pod+service and the cronJob. Put the service+pod into the `pods/smoke-test.yaml` and create a separate `jobs/smoke-test.yaml`